src/v/raft/group_configuration.h

```diff
@@ -0,0 +1,323 @@
+/*
```

Comment:
@96RadhikaJadhav did you use `git mv <source> <destination>` ? i don't see why we should have so many additional lines in the git. 

also, i think you'll need to actually compile the code and fix the header imports.